File: front/frontApi/src/app/pageDoador/form-doador/doador.ts
import { Usuario } from "src/app/cadastro-do-usuario/usuario";

export interface Doador {
	id: number,
  	nomeCompleto: String,
	cep: String,
	logradouro: String,
	numeroCasa: number,
	bairro: String,
	cidade: String,
	estado: String,
	complemento: String,
	telefone: String,
	celular: String,
	termo: String,
	usuario: Usuario 
  }

File: back2/doaju/src/main/java/br/com/doaju/model/Doador.java
package br.com.doaju.model;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.fasterxml.jackson.annotation.JsonManagedReference;

import lombok.Data;

@Data
@Entity
public class Doador implements Serializable{
	
private static final long serialVersionUID = 1L;
	
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private Long id;
	
	private String nomeCompleto;	
	private String cep;
	private String logradouro;
	private int numeroCasa;
	private String bairro;
	private String cidade;
	private String estado;
	private String complemento;
	private String telefone;
	private String celular;
	private boolean termo;
	
	@Temporal(TemporalType.TIMESTAMP)     
	private Date dataCadastro = new java.sql.Date(System.currentTimeMillis());
	
	@ManyToOne
	@JsonManagedReference
	@JoinColumn(name = "usuario_id", nullable = false)
	private Usuario usuario;
	

}
